### DISCUSSION TITLE
Please add support for Scaleway Object Storage As 75 gb is free for life

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#4123] [Self Hosting]
Please can you add scaleway s3 storage to outline ??
It would be helpful for students 

https://www.scaleway.com/en/pricing/?tags=storage

**Comment by Limezy** [comment]
If Scaleway free storage is s3 compatible, then it *should* work with Outline.
The only way to know is probably to test it ! https://app.getoutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e/doc/aws-s3-N4M0T6Ypu7
As of now, Amazon, MinIO and Digital Ocean are known to work, Blackblaze is known not to work ([because they don't support uploading via POST](https://github.com/outline/outline/discussions/2666))
